src/pyroboplan/models/two_dof.py

```diff
@@ -20,3 +23,46 @@ def load_models():
     urdf_filename = os.path.join(package_dir, "two_dof.urdf")
 
     return pinocchio.buildModelsFromUrdf(urdf_filename, package_dirs=models_folder)
+
+
+def add_object_collisions(model, collision_model, visual_model):
+    """
+    Adds obstacles and collisions to the 2-DOF manipulator collision model.
+
+    Parameters
+    ----------
+        model : `pinocchio.Model`
+            The Panda model.
```

Comment:
Lol this doesn't really matter but s/Panda/2-DOF/

Reply:
eagle eye!